src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +50,302 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
```

Comment:
Maybe 'substitude' is very generic for a module name... what about macroSubscritions ?